src/form/form.tsx

```diff
@@ -37,6 +37,16 @@ const getDeclaredFields = (groups: Array<Group>) => {
     );
 };
 
+const getDeclaredVisibleFields = (groups: Array<Group>) => {
```

Comment:
Can't you just use `getDeclaredFields()` and apply the visible filter to that result?

Reply:
It's a map and deep filtering will require more lines of code than one; looked ugly when i tried it. Plus the extra declarative name of the method is really helpful.

Reply:
Ok I got what you mean :) Done.